runtimes/runtimes/operational-telemetry/telemetry-schemas/server-crash-schema.json

```diff
@@ -0,0 +1,20 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "definitions": {
+        "ServerCrashEvent": {
+            "title": "ServerCrashEvent",
+            "allOf": [
+                { "$ref": "./base-event-schema.json" },
+                {
+                    "required": ["crashType"],
+                    "additionalProperties": false,
+                    "properties": {
+                        "crashType": {
+                            "type": "string"
```

Comment:
I'm thinking it will be pretty difficult to troubleshoot a crash based on only the type of crash. I think we need a way to log related error stacktrace and error message. I do not think it should obligatory be part of metrics - it's rather logging details, but we should design this as well.

Reply:
We have a separate task for figuring out that part.